pkgs/applications/misc/flashprint/default.nix

```
@@ -0,0 +1,34 @@
+{ stdenv, lib, libGLU, qtbase, fetchurl, dpkg, autoPatchelfHook, wrapQtAppsHook }:
+
+let
+  version = "5.7.1";
+
+in stdenv.mkDerivation {
+  pname = "flashprint";
+  inherit version;
+
+  src = fetchurl {
+    url = "http://www.ishare3d.com/3dapp/public/FlashPrint-5/FlashPrint/flashprint5_${version}_amd64.deb";
+    hash = "sha256-kxvqEgXlKQlfzlCqKb5o3hvop82vDsJmQDK9XOCq61g=";
+  };
+
+  nativeBuildInputs = [ dpkg autoPatchelfHook wrapQtAppsHook ];
+
+  buildInputs = [ qtbase libGLU ];
+
+  installPhase = ''
```

```suggestion
  buildInputs = [ qtbase libGLU ];

  qtWrapperArgs = [ "--prefix QT_QPA_PLATFORM : xcb" ];

  installPhase = ''
```
Without this, it's broken on Wayland. See the discussion on #241474.

---

Thanks!